nixos/modules/services/misc/paperless-ng.nix

```diff
@@ -298,6 +309,7 @@ in
         group = defaultUser;
         uid = config.ids.uids.paperless;
         home = cfg.dataDir;
+        extraGroups = [ config.services.redis.servers.paperless-ng.user ];
```

Comment:
Just tried it out locally, where I run paperless as my user, so I can copy files into the consumption directory, so this line didn’t get applied.

What do you think of adding the group as `SupplementaryGroups = [ config.services.redis.servers.paperless-ng.user ];` to the systemd services instead of here?

Reply:
I agree, this should set `SupplementaryGroups` instead.

Reply:
Sorry about the delay, will fix that tomorrow :)

Reply:
Should be fixed!

Reply:
Did you push your changes? :)

Reply:
whoops sorry, done